docs/release_guide.rst

```diff
@@ -21,8 +26,17 @@ that everything you need to perform the release is correctly installed
 and that you have the correct permissions. All rever commands should be
 run in the root level of the repository.
 
+..
+    rever currently is set to upload to PyPI, which will likely cause issues as
+    we need to upload the sdist and wheels at the same time
```

Comment:
Do we? I thought it was possible to upload them separately, so long as each file is only uploaded once.

Reply:
https://github.com/h5py/h5py/issues/1300#issuecomment-531407658 was what I was going off, I think the issue is if there's no wheel for the latest version, pip tries to build from source, which errors, meaning the install fails. I guess it depends on how long it takes to get and upload the wheels (the longer it is, the more people get a build error - I imagine a hour or so it would be fine, longer I'm not so sure).

Reply:
Ah, gotcha. I was thinking of limitations on uploading to PyPI, but it makes sense that we don't want a release uploaded without wheels for any length of time.